### ISSUE TITLE
Accessibility: Insufficient color contrast of some elements

### ISSUE THREAD

**Issue by dgodongm** [mastodon/joinmastodon#362]
Text on numerous buttons and a paragraph element have insufficient color contrast against their background. 

Repro Steps:
1. Navigate to https://joinmastodon.org/
2. Run Axe DevTools or other A11Y tools

Results:
9 elements flagged as having insufficient color contrast: 8 buttons which are only slightly below WCAG 2 AA 4.5:1 guidance at 4.37:1; and a <p> element with text that only has 2.77:1

![Screenshot 2023-06-27 at 12 58 03 PM](https://github.com/mastodon/joinmastodon/assets/123777263/2ee66b4f-4839-467b-abdb-a03004ef777b)

Expected Results:
All elements should meet WCAG 2 AA guidance on color contrast to better ensure content is accessible

**Comment by dgodongm** [comment]
Similar/same issues occur throughout https://joinmastodon.org/ pages

**Comment by dgodongm** [comment]
Seems addressing this may be challenging given Mastodon's [branding colors](https://joinmastodon.org/branding)

**Comment by dgodongm** [comment]
@alireza-hariri The text noted in the issue is 16px
